test/views/MakingALateAppealPageSpec.scala

```diff
@@ -33,7 +34,8 @@ class MakingALateAppealPageSpec extends SpecBase with ViewBehaviours {
     }
     def applyView(form: Form[_]): HtmlFormat.Appendable = makingALateAppealPage.apply(form)
 
-    val formProvider = MakingALateAppealForm.makingALateAppealForm
+    val formProvider = MakingALateAppealForm.makingALateAppealForm(fakeRequest.
+      withSession(SessionKeys.reasonableExcuse -> "crime"))
```

Comment:
Should be able to revert this if the custom error message is not needed

Reply:
Yes custom message not needed, hence removed.